amical/tests/test_tools.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+
+from amical import tools
+
+
+def test_crop_max():
+
+    img_size = 80  # Same size as NIRISS images
+    img = np.ones((img_size, img_size))
+    xmax, ymax = 17, 57
+    img[ymax, xmax] = img.max() * 5  # Add off-centered max pixel
```

Comment:
this looks like a very particular case, maybe it'd make a more robust test if the image was random. You can then manually set the center to the lowest value since it's important for this test that the max be off-centered

Reply:
Do  you mean making the whole image random instead of ones and one value > 1 ? Or do you mean making the max position random? I'm not sure what you by "manually set the center to the lowest value". Also, after looking into it a bit more, it should not matter that the location is off-centered, as long as the cropped size is bigger than the distance to the edges.

Reply:
I made the test random in the last commit. Should I use [`pytest-repeat`](https://github.com/pytest-dev/pytest-repeat) to repeat the test a few times to make it more robust ?